patches/host-spawn.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/go.mod b/go.mod
```

Comment:
Security vulnerability patch. Waiting for owner review - https://github.com/1player/host-spawn/pull/30.